common/web/keyboard-processor/src/keyboards/keyboard.ts

```diff
@@ -31,29 +31,152 @@ export interface VariableStoreDictionary {
+   *  within app/webview.
+   */
   KI: string;
+  /**
+   * Keyboard Name:  the human-readable name of the keyboard.
+   */
   KN: string;
+  /**
+   * Encoded data usable to construct a desktop/hardware-oriented on-screen keyboard.
+   */
   KV: EncodedVisualKeyboard;
+  /**
+   * Keyboard Language Code: set within select keyboards.
+   *
+   * Currently, it's only used to determine the need for CJK-picker support.  Is missing
+   * in most compiled keyboards.
+   */
   KLC?: string;
+  /**
+   * @deprecated
+   * Keyboard Language Code: set within select keyboards.
+   *
+   * Currently, it's only used to determine the need for CJK-picker support.
+   * Is (probably) an older name of KLC with the identical purpose.  Is missing
+   * in most compiled keyboards.
+   */
   LanguageCode?: string;
+  /**
+   * Keyboard CSS: provides the definition for custom keyboard style sheets
+   */
   KCSS?: string;
+  /**
+   * Keyboard is RTL: a simple flag noting if the keyboard's script is RTL.
+   */
   KRTL?: boolean;
+  /**
+   * Keyboard Modifier BitMask:  a set of bitflags indicating which modifiers
+   * the keyboard's rules utilize.  See also: `Codes.modifierCodes`.
+   */
   KMBM?: number;
+  /**
+   * Keyboard Supplementary plane:  set to 1 if the keyboard uses non-BMP Unicode
+   * characters.
+   */
   KS?: number;
+  /**
+   * Keyman Visual Keyboard Layout:  defines the touch-layout definitions used for
+   * 'phone' and 'tablet' form-factors.
+   */
   KVKL?: LayoutSpec;
+  /**
+   * Keyboard is Mnemonic: set to 1 if the keyboard uses a mnemonic layout.
+   */
   KM?: number;
+  /**
+   * KeyBoard VERsion: the version of this keyboard.
+   */
   KBVER?: string;
+  /**
+   * Keyman VERsion:  the version of Keyman Developer used to compile this keyboard.
+   */
   KVER?: string;
+  /**
+   * Keyman Variable Stores: an array of the names of all variable stores used by the
+   * keyboard.
+   */
   KVS?: (`s${number}`)[];
+  /**
+   * Keyboard Help: HTML help text, as specified by either the &kmw_helptext or &kmw_helpfile system stores.
+   *
+   * Reference: https://help.keyman.com/developer/language/reference/kmw_helptext,
+   *            https://help.keyman.com/developer/language/reference/kmw_helpfile
+   */
   KH?: string;
+  /**
+   * Keyboard Virtual Key Dictionary: the Developer-compiled, minified dictionary of virtual-key codes
+   */
   KVKD?: string;
+  /**
+   * Keyboard Display Underlying:  set to 1 if the desktop form of the keyboard
+   * should show the US QWERTY underlying keycaps.  These may also appear on
+   * touch layouts if set and no touch-layout information is available.
+   */
   KDU?: number;
+  /**
+   * Virtual Key Dictionary: the engine pre-processed, unminified dictionary.  This is built within
+   * Keyman Engine for Web at runtime as needed based on the definitions in `KVKD`.
+   */
   VKDictionary?: Record<string, number>,
```

Comment:
I reckon we should put `VKDictionary` and `_kmw` into a subclass, as they are internal to Keyman Engine for Web. But that can wait until we move to common/web/types.